app/views/DocumentEditView/DocumentEditView.css

```diff
@@ -1,3 +1,7 @@
+.main {
+  padding-bottom: calc(var(--editif-toolbar-height) + var(--edit-margin-md));
```

Comment:
Hmm, this grey gap seems unwholesome to me 

![screenshot-patch-mobile-fixes publish dadi technology-2019 09 22-11_06_00](https://user-images.githubusercontent.com/21290049/65384959-fe83ed80-dd28-11e9-8eca-8ba1d95fcc2c.png)